site/content/docs/5.2/migration.md

```diff
@@ -19,6 +19,31 @@ If you need more details about the changes, please refer to the [v5.2.3 release]
 
 ### Components
 
+- **Cards**
```

Comment:
It's not really the components that have been modified but the example page so this modification should rather appear in an "Examples" section.

Instead of "We had an issue with some `<a>` that missed `[href]` attribute on cards example." rather say something that those `<a>`s weren't links but buttons so should use `<button>`s HTML element. And a "please reflect" sentence as usual.